README.md

```diff
@@ -253,7 +253,7 @@ login:
 
 #### 2) Ad configuration
 
-Each ad is described in a separate JSON or YAML file.
+Each ad is described in a separate JSON or YAML file with prefix `ad_<filename>`.
```

Comment:
This is actually only the default and can be overridden in the config.yaml Would you mind adding this?

Reply:
done